### ISSUE TITLE
booktitle not exported with export_bibtex

### ISSUE THREAD

**Issue by hellemo** [Humans-of-Julia/Bibliography.jl#15]
Great job with Bibliography.jl!

Some of my references don't survive the roundtrip from bibtex via Bibliography.jl and back to bibtex, it seems like the booktitle is lost during the export_bibtex:

```julia
using Bibliography

testdata = """@inproceedings{demo2020proceedings,
 organization  = {DemoOrg},
 pages         = {1--10},
 doi           = {10.1000/001-1-001-00001-1_001},
 author        = {Demo, D},
 note          = {cited by 0},
 year          = {2020},
 booktitle     = {Demo Booktitle},
 title         = {Demo Title}
}"""

write("demo.bib",testdata)
mybib = Bibliography.import_bibtex("demo.bib")
Bibliography.export_bibtex("demo_export.bib",mybib)
mybib2 = Bibliography.import_bibtex("demo_export.bib")
```

gives the error:
```
mybib2
Entry demo2020proceedings is missing the booktitle field(s).

error(::String)@error.jl:33
make_bibtex_entry(::String, ::Dict{String, String})@bibtex.jl:81
parse_file(::String)@bibtex.jl:174
#parse_file#1@BibParser.jl:19[inlined]
parse_file@BibParser.jl:18[inlined]
import_bibtex(::String)@bibtex.jl:6
top-level scope@Local: 1[inlined]
```

**Comment by Azzaare** [comment]
Sorry @hellemo ,

For some reason, I missed the notifications for a long period. I am (slowly because of work constraints) rewriting the bibtex parser in BibParser.jl. Once it is done, I will go through the different issues related to Bibliography.jl, including yours.

**Comment by Azzaare** [comment]
This was solved in the latest release (2.7)
The booktitle field was simply not exported in Bibliography.jl when exporting to BibTeX.

Thanks for catching the bug @hellemo
